src/FinalForm.ts

```
@@ -225,12 +228,13 @@ function createForm<
   const scheduleAsyncCallbacks = () => {
     if (!asyncCallbacksScheduled && pendingAsyncCallbacks.length > 0) {
       asyncCallbacksScheduled = true;
-      const scheduler = callbackScheduler || ((callback) => setTimeout(callback, 0));
+      const scheduler =
+        callbackScheduler || ((callback) => setTimeout(callback, 0));
       scheduler(() => {
         const callbacks = [...pendingAsyncCallbacks];
         pendingAsyncCallbacks.length = 0;
         asyncCallbacksScheduled = false;
-        callbacks.forEach(callback => callback());
+        callbacks.forEach((callback) => callback());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)